fast_versioning.gemspec

```diff
@@ -13,8 +13,8 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,db,lib}/**/*', 'Rakefile', 'README.md']
 
-  s.add_dependency 'rails', '>= 5.0', '< 5.2'
-  s.add_dependency 'paper_trail', '>= 4.2', '< 6'
+  s.add_dependency 'rails', '>= 5.0', '< 5.3'
+  s.add_dependency 'paper_trail', '>= 4.2', '< 10.4'
```

Comment:
It doesn't feel super intuitive to have these upper bound specifiers be "the version after the latest version we support". Is there a better way of doing it?

Reply:
we can just remove upper bond if we know it's compatible with the latest and just add if the issue is reported with a newer version later

Reply:
Sounds good, I'll remove them.